### PR TITLE
Improve XSS sanitizing of `Text` fields

### PR DESCRIPTION
**Pull request by nikku**
I've decided for the approach "improve our existing sanitizer" as outlined https://github.com/bpmn-io/form-js/commit/46357a6e4adcadec06e0d5a791f3e3efd61c3350.

Closes https://github.com/bpmn-io/form-js/issues/107.

### CODE REVIEW

**Comment by nikku** [comment]
Fun fact: The huge bahttps://cure53.de/purifyd HTML snipped provided [on the DOMPurify website](https://cure53.de/purify) does not parse at all in Firefox. I added additional precautions to ensure that we don't fail in unparseable HTML cases, too: https://github.com/bpmn-io/form-js/pull/108/commits/0abd9244092979d3f408a93550b04b41c39e402a.

File: packages/form-js-viewer/src/render/components/Sanitizer.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+const ALLOWED_ATTRIBUTES = [
+  'align',
+  'alt',
+  'class',
+  'href',
+  'id',
+  'name',
+  'src',
+  'valign'
```
Comment (barmac) [inline packages/form-js-viewer/src/render/components/Sanitizer.js:35]: What is this for since we don't allow `<table>` and friends?

**Review by philippfromme — APPROVED** [review]
In the future we could think about allowing tables. We have no styling for table anyway at the moment so I think it's good that we disallow them for now.

Substantial improvement! 👏🏻

File: packages/form-js-viewer/src/render/components/Sanitizer.js (hunk 1 above)
Comment (philippfromme) [reply packages/form-js-viewer/src/render/components/Sanitizer.js:35]: Oh. Sorry, I just added a comment about tables. 😅

Comment (nikku) [reply packages/form-js-viewer/src/render/components/Sanitizer.js:35]: Removed via https://github.com/bpmn-io/form-js/commit/2fe2b202ca2c59588495931cefc3c886af907545.

**Pull request merged**